src/ControlzEx.Showcase/Themes/Controls.xaml

```diff
@@ -1,18 +1,16 @@
 <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:controlzex="urn:controlzex">
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
     <Style TargetType="AccessText">
         <Setter Property="OverridesDefaultStyle" Value="True" />
     </Style>
 
     <Style TargetType="Label">
         <Setter Property="OverridesDefaultStyle" Value="True" />
-        <Setter Property="Padding"
-                Value="5" />
-        <Setter Property="HorizontalContentAlignment"
-                Value="Left" />
-        <Setter Property="VerticalContentAlignment"
-                Value="Top" />
+        <Setter Property="Padding" Value="5" />
+        <Setter Property="HorizontalContentAlignment" Value="Left" />
+        <Setter Property="VerticalContentAlignment" Value="Top" />
+        <Setter Property="RenderOptions.ClearTypeHint" Value="Enabled" />
```

Comment:
We shouldn't set those here.
We should better add options to the showcase app for those.